gnovm/tests/imports.go

```diff
@@ -413,6 +418,10 @@ func TestStore(rootDir, filesPath string, stdin io.Reader, stdout, stderr io.Wri
 			stdlibPath := filepath.Join(rootDir, "gnovm", "stdlibs", pkgPath)
 			if osm.DirExists(stdlibPath) {
 				memPkg := gno.ReadMemPackage(stdlibPath, pkgPath)
+				if memPkg.IsEmpty() {
+					panic(fmt.Sprintf("found an empty package %q", pkgPath))
+				}
```

Comment:
It's probably out of the scope of this PR, but what do you think about refactoring the `TestStore` method to take in a `testing.T`, indicating it should be used only for tests?
I see it's used for major code flows (outside of tests), so it would require a larger cleanup effort. Overall I think we need to rethink how this method is being used, since it can panic out during `gno run` or `gno test`

Reply:
(No need to change it here in the PR, you can resolve this convo)